src/Sylius/Behat/Context/Cli/ShowingAvailablePluginsContext.php

```
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Cli;
+
+use Behat\Behat\Context\Context;
+use Sylius\Bundle\CoreBundle\Command\SetupCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Webmozart\Assert\Assert;
+
+final class ShowingAvailablePluginsContext implements Context
+{
+    /** @var KernelInterface */
+    private $kernel;
+
+    /** @var Application */
+    private $application;
+
+    /** @var CommandTester */
+    private $tester;
+
+    /** @var SetupCommand */
+    private $command;
+
+    public function __construct(KernelInterface $kernel)
+    {
+        $this->kernel = $kernel;
+    }
+
+    /**
+     * @When I run show available plugins command
+     */
+    public function runShowAvailablePluginsCommand(): void
+    {
+        $this->application = new Application($this->kernel);
+        $this->application->add(new SetupCommand());
+
+        $this->command = $this->application->find('sylius:show-available-plugins');
+        $this->tester = new CommandTester($this->command);
+
+        try {
+            $this->tester->execute(['command' => 'sylius:show-available-plugins']);
+        } catch (\Exception $e) {
```

Should we silence the exception? And `$e` -> `$exception` :)